seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java

```diff
@@ -73,6 +73,24 @@ protected final void prepare(E env, List<? extends Plugin<E>>... plugins) {
         }
     }
 
+    /**
+     * Execute close method defined in {@link org.apache.seatunnel.plugin.Plugin}
+     *
+     * @param plugins plugin list
+     */
+    @SafeVarargs
+    protected final void close(List<? extends Plugin<E>>... plugins) {
+        for (List<? extends Plugin<E>> pluginList : plugins) {
+            pluginList.forEach(plugin -> {
+                try {
+                    plugin.close();
```

Comment:
will close() be implemented to do resource clean? are you assume that if one plugin throw exception when call close(), the followed plugins no need to call close()?
maybe we should add some comment to stress what it shouldn't do.

Reply:
@yx91490 Yes, close is mainly used to do resource clean, this is a good catch, I have added some comment in `close` method.
We have another choose to catch the exception and make it safe to close for each plugin, but I think throw it directly is OK.